generatedocs/data/params.json

```diff
@@ -137,6 +137,11 @@
         "`true`: Every iteration for each concurrent user uses the same user and session.",
         "`false`: Every iteration for each concurrent user uses a new user and session. The total number of users is the product of `concurrentusers` and `iterations`."
     ],
+    "config.scheduler.settings.onlyinstanceseed": [
+        "Disable session part of randomization seed. Defaults to `false` if omitted.",
```

Comment:
Slight rewrite suggested:
"Disable the session part of the randomization seed. Defaults to 'false', if omitted."